From 91666df0363161f02dfdc40124a3a7fa1fd3918d Mon Sep 17 00:00:00 2001
From: Emilio Jesus Gallego Arias <e+git@x80.org>
Date: Thu, 4 Mar 2021 20:10:06 +0100
Subject: [PATCH] [coq] Overlay for coq/coq#13617

You may likely want to make this change conditional on Coq's version,
but it is out of my autoconf skills.
---
 Remakefile.in | 2 +-
 configure.in  | 4 ++--
 2 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/Remakefile.in b/Remakefile.in
index 6312b69..4fb45b5 100644
--- a/Remakefile.in
+++ b/Remakefile.in
@@ -43,7 +43,7 @@ configure config.status: configure.in
 	@COQC@ @COQEXTRAFLAGS@ -R src Gappa -I src $<
 
 COQPKGS = clib engine kernel interp lib library parsing pretyping printing proofs tactics toplevel vernac plugins.ltac
-PACKAGES = $(addprefix -package coq., $(COQPKGS)) -package @BIGINTPKG@
+PACKAGES = $(addprefix -package coq-core., $(COQPKGS)) -package @BIGINTPKG@
 
 src/gappatac.ml: src/gappatac.c
 	@CPP@ -DCOQVERSION=@COQVERSION@ $< -o $@
diff --git a/configure.in b/configure.in
index 8dd74bd..5cbaba2 100644
--- a/configure.in
+++ b/configure.in
@@ -88,7 +88,7 @@ if test "$native_tactic" = yes; then
   AC_MSG_CHECKING([for native development files])
   AS_IF(
     [ echo "let _ = (Evd.empty, $BIGINTMOD.zero_big_int)" > conftest.ml
-      $OCAMLFIND ocamlopt -rectypes -thread -package coq.engine -package $BIGINTPKG -shared conftest.ml -o conftest.cmxs ],
+      $OCAMLFIND ocamlopt -rectypes -thread -package coq-core.engine -package $BIGINTPKG -shared conftest.ml -o conftest.cmxs ],
     [ AC_MSG_RESULT(yes) ],
     [ AC_MSG_RESULT(no)
       AC_MSG_ERROR([ *** Failed to compile a native OCaml library ]) ])
@@ -98,7 +98,7 @@ if test "$byte_tactic" = yes; then
   AC_MSG_CHECKING([for bytecode development files])
   AS_IF(
     [ echo "let _ = (Evd.empty, $BIGINTMOD.zero_big_int)" > conftest.ml
-      $OCAMLFIND ocamlc -rectypes -thread -package coq.engine -package $BIGINTPKG -c conftest.ml -o conftest.cmo ],
+      $OCAMLFIND ocamlc -rectypes -thread -package coq-core.engine -package $BIGINTPKG -c conftest.ml -o conftest.cmo ],
     [ AC_MSG_RESULT(yes) ],
     [ AC_MSG_RESULT(no)
       AC_MSG_ERROR([ *** Failed to compile a bytecode OCaml library ]) ])
-- 
2.24.3 (Apple Git-128)

